src/System.Security.Cryptography.Algorithms/tests/SignatureDescriptionTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// (C) 2002 Motus Technologies Inc. (http://www.motus.com)
+// (C) 2004 Novell  http://www.novell.com
+
+using Xunit;
+
+namespace System.Security.Cryptography.Algorithms.Tests
+{
+    public class SignatureDescriptionTests
+    {
+        //protected SignatureDescription sig;
+        //protected static DSA dsa;
+        //protected static RSA rsa;
+
+        //public SignatureDescriptionTests()
+        //{
+        //    sig = new SignatureDescription();
+        //    // key generation is VERY long so one time is enough
+        //    if (dsa == null)
+        //        dsa = DSA.Create();
+        //    if (rsa == null)
+        //        rsa = RSA.Create();
+        //}
+
+        [Fact]
+        public void Constructor_Default()
+        {
+            SignatureDescription sig = new SignatureDescription();
```

Comment:
None. I'm refactoring it to do the default value checks in this method instead of Properties.